weTranslate/NoResultView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  NoResultTableViewCell.swift
+//  weTranslate
+//
+//  Created by Lionel on 2/28/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import UIKit
+
+final class NoResultView: UIView {
+
+    // MARK: - Properties
+
+    var viewModel: NoResultViewModel? {
+        didSet {
+            noResultLabel.text = viewModel?.message
+            emojiLabel.text = viewModel?.emoji
```

Comment:
No, we need to update it for each search. It's nil before the first search.
We could do it by creating a new `NoResultView` for each research and add it to the stack view then, but I don't think it's worth it.